src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameTests.cs

```diff
@@ -141,9 +146,9 @@ public void OptionSetDisplayNames(string inputExpression, string outputExpressio
             }
             else
             {
-                var outInvariantExpression = engine.GetInvariantExpression(outputExpression, new RecordType());
+                var outInvariantExpression = engine.GetInvariantExpression(inputExpression, new RecordType());
```

Comment:
>inputExpression [](http://example.com/codeflow?start=75&length=15)

double checking , outputExpression --> inputExpression?

Reply:
Yeah, this test was accidentally passing previously, it wasn't testing the right thing. It was actually testing that passing an invariant expression to GetInvariantExpression returns the same invariant expression.